helpers/log_helper.py

```diff
@@ -0,0 +1,40 @@
+_REDSHIFT_MISMATCH_LOG = (
+    'Number of {database_type} records does not match number '
+    'of Redshift {redshift_table} records: {database_count} '
+    '{database_type} records and {redshift_count} Redshift records'
+)
+
+_NO_RECORDS_FOUND_LOG = (
+    'No {database_type} records found for all of {date}'
+)
+
+_SIERRA_NULL_CODES_LOG = (
+    'The following {code_type} have a null value for one or more of '
+    'their inferred columns: {codes}'
+)
+
+_SIERRA_DUPLICATE_CODE_LOG = (
+    'Duplicate {code_type} codes found in Redshift: {total_count} '
+    'total active {code_type} codes but only {distinct_count} '
+    'distinct active {code_type} codes'
+)
+
+def build_redshift_mismatch_log(database_type, redshift_table,
```

Comment:
I was thinking this file would also include the alarm logic and would actually do the logging. If we're already passing in the counts we as might as well do the conditional here as well. What do you think?

Reply:
Yeah that's very doable! Let me make those adjustments